### ISSUE TITLE
How to set base terrain texture in v2.0.0?

### ISSUE THREAD

**Issue by heath3n** [mapbox/mapbox-unity-sdk#1129]
I just upgraded from 1.4.4 and my ground is now bright pink due to the base terrain texture option being removed (or maybe moved and I just can't find it). Looking in Studio, it appears that the layer I want to texture should be `background` but lo and behold the feature modifier input has also been changed and is now a dropdown from which the `background` layer is absent.

So how can I add a texture that covers the entire map?

**Comment by atripathi-mb** [comment]
@heath3n the `Terrain Material` is deprecated and now you can set a default material using `Tile Material` under the `General` section.


I’m not sure what you mean by `feature modifier input has been changed `? Could you give us more details?

**Comment by heath3n** [comment]
Yeah, sorry that was unclear. Under `Features` in the map component is the option to add visualizers to a certain data layer. The last time I checked that section (which was apparently earlier than in 1.4.4) it was an input field that allowed us to type anything in there (I think the type e.g. `Point`, `Line`, etc. was also open-ended?). Now I find it to be a dropdown, so it seems styling other layers is not supported (or maybe it never was).
